website/api.php

```diff
@@ -246,7 +246,7 @@ function validateSetupWB($setup,$hero,$fols,$mode) {
         echo json_encode(array("success"=>false,"error"=>"Too much followers"));
         return false;
     }
-    if ($units>6) {
```

Comment:
can this even be above 6? Isn't it only incremented in the for-loop starting a line 215, so a maximum of 6 times?

Reply:
Agreed